librz/main/rz-ax.c

```diff
@@ -736,3 +736,37 @@ RZ_API int rz_main_rz_ax(int argc, const char **argv) {
 	num = NULL;
 	return 0;
 }
+
+char *rz_ax_parse_bitstring(const char *bitstring) {
+	// Check that bitstring starts with "0b"
+	if (strncmp(bitstring, "0b", 2) != 0) {
+		RZ_LOG_ERROR("rz-ax: bitstring must start with '0b'\n");
+		return NULL;
+	}
+	// Get the length of the bitstring
+	size_t bitlen = strlen(bitstring) - 2;
+	// Check that bitstring is "0b" only 
+	if (!bitlen) {
+		RZ_LOG_ERROR("rz-ax: bitstring is too short\n");
+		return NULL;
+	}
+	// Create a bitvector of the appropriate size
+	RzBitVector *bitv = rz_bv_new(bitlen);
```

Comment:
as i mentioned before, if bitlen is 0, we have an issue, could you add a check to be sure it is always at least of the size of `1`? the check should be about ensuring the header starts either with `0b0` or `0b1`, since those are edge cases.

Reply:
@wargio please check lines from 748 to 752

Reply:
ops i missed that.